example/.eslintrc

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
Agreed. This eslintrc is fine for the example. Let's improve the eslintrc that Gadget provides.